src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -819,8 +819,8 @@ def _show_app(cmd, resource_group_name, name, cmd_app_type, slot=None):
     app_type = _kind_to_app_type(app.kind) if app else None
     if app_type != cmd_app_type:
         raise ResourceNotFoundError(
-            "Unable to find {} '{}', in RG '{}'".format(cmd_app_type.value, name, resource_group_name),
-            "Use 'az {} show' to show {}s".format(app_type.value, app_type.value))
+            "Unable to find {} '{}', in RG '{}'".format(cmd_app_type, name, resource_group_name),
+            "Use 'az {} show' to show {}s".format(app_type, app_type))
```

Comment:
Need fix Style Check problem：
`"Use 'az {0} show' to show {0}s".format(app_type))`
or
`"Use 'az {app_type} show' to show {app_type}s".format(app_type=app_type))`